WG_JS_CORE.md

```diff
@@ -50,7 +52,7 @@ JS IPFS has for a long time now been under heavy development with a focus on imp
 - An IPFS log API is implemented and tests ensure log messages are part of the API
 - A security audit is conducted on the code base and all issues resolved
 
-### `M (P1)`: Optimized for browser environments 🔄 🤝 🧠
+### `M (P1)`: Optimized for browser environments 📦 🤝
```

Comment:
This Milestone will enable the Web Browsers Working Group to ship an amazing DWebCDN using JS IPFS and given that the `<script> tag` is still the most widely used package manager of the Web, it will have a tremendous impact.

Consider dropping priority for Electron, Mobile and/or IoT